### DISCUSSION TITLE
How to understand three local wind speed concepts in the Aerodyn OutList.

### DISCUSSION THREAD

**Discussion by ietqlw** [OpenFAST/openfast#1028] [Q&A] [answered]
Hello everyone,

        In the Aerodyn manual, three local wind speed confused me a lot. Would you mind sharing me your views about their differences?
        (1) Undisturbed wind velocity at BαNβ in the local blade coordinate system.
        (2) Disturbed wind velocity at BαNβ in the local blade coordinate system.
        (3) Relative wind speed at BαNβ.
![微信图片_20220305233615](https://user-images.githubusercontent.com/79843558/156890067-e1395cf7-7fb4-4a93-acc4-1e044cef5d2c.png)

**Comment by ebranlard** [comment, marked as answer]
- The undisturbed wind velocity (`VUnd`) is the wind speed returned by inflow wind. It will contain turbulence, shear (or deterministic wind) but nothing else. 
- The disturbed wind velocity (`VDis`) is the undisturbed wind velocity (`VUnd`) plus the disturbances associated with the tower shadow and tower potential model. It does not contain any induction effects from the rotor. 
- `STV` is the structural velocity of the blade element
- `VInd` is the induced velocity at the blade element. 
- `Vx` and `Vy`  (not in your list above) represent the relative flow velocity without induction: `VDis - STV`. 
- The relative velocity perceived by a blade element is the sum of the disturbed wind and induced velocity minus the structural velocity, that is: `VDis+VInd- STV`.  Only the `x-y` components are considered for the relative velocity since each blade element is assumed to act as a 2D airfoil.  We do not output the `x` and `y` components of the relative velocity (it can be computed from the other outputs), but the output channel `Vrel` is the norm of the relative velocity in the `x-y` plane. 

The figure below illustrates the `x-y` plane:










![AeroDynCoord](https://user-images.githubusercontent.com/1318316/157079591-6251bbdc-5260-4798-a929-e92d32779a9a.png)





There are some important differences when OpenFAST is used with a CFD solver (`CompInflow=2`).  `VUnd` will contains the inflow and induced velocity, therefore `Vx` will also contain the induced velocities, but the variable `VInd` will be zero. The reason is that in general it is not possible to dissociate between inflow and induction in a CFD software (unless uniform inflow is used).

**Comment by ietqlw** [comment]
Thank you for your careful answer.
